charts/gateway-crds-helm/values.yaml

```
@@ -0,0 +1,5 @@
+crd:
+  apiGateway:
+    enabled: false
+  envoyGateway:
+    enabled: true
```

- `crd` -> `crds` since there are multiple
- `apiGateway` -> `gatewayApi` since that's the correct name
```suggestion
crds:
  gatewayApi:
    enabled: false
  envoyGateway:
    enabled: true
```

---

Sure, I will make it enabled by default